css-contain-2/Overview.bs

```diff
@@ -1654,6 +1647,32 @@ Suppressing An Element's Contents Entirely: the 'content-visibility' property {#
 			(or within the user-agent defined margin around the viewport),
 			even if the element itself is still off-screen.
 
+
+		* The element is <dfn>far away from the viewport</dfn>:
+			In this state, the element's proximity to the viewport has been computed and is not [=close to the viewport=].
+		
+		* The element's [=proximity to the viewport=] is not determined:
+			In this state, the computation to determine the element's [=proximity to the viewport=] has not been done.
+		
+		By default, the element's [=proximity to the viewport=] is not determined.
```

Comment:
I wonder if "Initially" or "When an element is created" would be clearer than "By default"

Reply:
Done.